packages/fhir-qb/index.js

```diff
@@ -451,15 +451,6 @@ class QueryBuilder {
         tokenQuery = this.qb.buildAndQuery(queries);
         break;
       case 'Identifier':
```

Comment:
Did you combined identifier and contact point into one case

Reply:
yes, it was just an exact copy